requirements/dev-local.txt

```diff
@@ -0,0 +1,3 @@
+ipython
+pdbr
+pytype
```

Comment:
I don't think this belongs in the shared dependencies as it is personal tooling.

Reply:
Right, missing from my global .gitignore